src/manifest.json

```diff
@@ -1,20 +1,22 @@
 {
   "name": "Dockstore",
   "short_name": "Dockstore",
-  "description": "Create, Share, Use",  
-  "icons": [{
-    "src": "assets/images/dockstore/dockstore.png",
+  "theme_color": "#21335B",
```

Comment:
Could you also explain these differences?

Also this is getting really nit-picky, but it would be nice if the diff showed just what you changed without having rearranged everything. If something goes wrong, it would be nice to tell what changed. I think it's that you added scope and start_url?

Reply:
Sorry about that, the committed `manifest.json` was auto-generated by Angular CLI when the service worker was installed. Since this worked with the prod build, I deleted the old manifest file and manually moved the important info to the new file

Reply:
Ah, if it's a generated file, then it may not be worth the effort